ansible/roles/operator-pipeline/templates/openshift/tasks/commit-pinned-digest.yml

```diff
@@ -3,21 +3,99 @@ apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: commit-pinned-digest
+  labels:
```

Comment:
We are not using labels for our tasks yet. However it's interesting idea, let's keep it consistent with existing ones :slightly_smiling_face: